qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -1,38 +1,45 @@
 - balloon_in_use:
     type = driver_in_use
-    only Windows
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
     repeat_times = 5
```

Comment:
is repeat_times=5 enouth for balloon in use test ?

Reply:
I mean linux guest.

Reply:
5 times is not enough for both linux and windows during_bg_test.
Modify to 100 to ensure guest is doing ballooning during background test.
Thanks